cmd/destroy.go

```diff
@@ -15,36 +15,72 @@
 package cmd
 
 import (
-	"fmt"
+	"strings"
+
+	"golang.org/x/net/context"
 
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/manifoldco/promptui"
+	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"fmt"
 )
 
-// destroyCmd represents the destroy command
 var destroyCmd = &cobra.Command{
 	Use:   "destroy",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Destroy the running KDK container",
+	Long:  `Destroy the running KDK container`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("destroy called")
+		logger := logrus.New().WithField("command", "destroy")
+
+		ctx := context.Background()
+		client, err := client.NewEnvClient()
+		if err != nil {
+			logger.WithField("error", err).Fatal("Failed to create docker client")
+		}
+
+		containers, err := client.ContainerList(ctx, types.ContainerListOptions{})
+
+		if err != nil {
+			logger.WithField("error", err).Fatal("Failed to list docker containers")
+		}
+
+		var containerIds []string
+
+		for _, container := range containers {
+			for _, name := range container.Names {
+				if strings.Contains(name, "kdk") {
```

Comment:
Should it be strings.StartsWith()?

Reply:
probably.  FYI go's docker sdk prefixes container names with `/`

So it would look like `if strings.HasPrefix(name, "/kdk") `